health-services/project-factory/src/server/api/genericApis.ts

```
@@ -1424,16 +1457,71 @@ function convertIntoSchema(data: any, isUpdate: boolean) {
   return data;
 }
 
+function convertIntoNewSchema(data: any) {
+  const properties: any = {};
+  const errorMessage: any = {};
+  const required: any[] = [];
+  let columns: any[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'columns' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)